### DISCUSSION TITLE
Ice Roadmap

### DISCUSSION THREAD

**Discussion by xjones** [jordanbaird/Ice#109] [General]
@jordanbaird just discovered Ice - looks great and love the simplicity. That said, in considering using Ice instead of Bartender there are a few features would love to have - e.g. permanent items, menubar layout and the ability to show overflow items beneath the main menubar. Would also be get some insight into your development roadmap. Keep up the great work.

**Comment by xjones** [comment]
I clearly missed the readme... so deleting this as there is a roadmap. If I have suggestions I'll create a more specific new discussion. thx.

**Comment by jordanbaird** [comment]
No worries! The roadmap is also pretty out of date. Put simply, I plan on making Ice (and/or a potential Ice Pro) one of the most powerful menu bars manager available, able to act as a drop-in replacement for Bartender. It will take some time -- Bartender has 10 years on Ice -- but I'm confident I can make it happen.
